### ISSUE TITLE
[Accessibility] - NVDA does not announce row or column headers

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4494]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Turn on NVDA.
2. Open https://ckeditor.com/ckeditor-4/demo/
3. Add a new table and add column headers for the table
4. Now using ctrl + alt + arrow keys navigate from one cell to another
5. Verify whether NVDA is announcing column headers or not.

### Expected result
NVDA should announce row or column headers when we navigate from one cell to another cell. When it going to the new cell it should announce the column headers of the new cell.

### Actual result
NVDA does not announce row or column headers.

## Other details
Attached the screen recording for clarity.

* Browser: Chrome
* OS: Windows 10
* CKEditor version: 4
* Installed CKEditor plugins: table
* Screen Reader: NVDA

[2137516_NVDA is not announcing headers as headers in table in CKEditor demo.zip](https://github.com/ckeditor/ckeditor4/files/5839218/2137516_NVDA.is.not.announcing.headers.as.headers.in.table.in.CKEditor.demo.zip)

**Comment by sculpt0r** [comment]
Hi, @sryarama ,

I'm sorry for the delay. Thank you for your engagement and effort to find and report this issue.  
  
I'm able to reproduce this issue.

**Comment by sryarama** [comment]
@sculpt0r, thank you for confirming. Do we have an ETA for fixing it?

**Comment by sculpt0r** [comment]
TBO - we just release the `4.16.0` version and we didn't schedule the next release and prioritize for it yet.

**Comment by sryarama** [comment]
Hi Team, is there any update on this? It is impacting our accessibility compliance.

**Comment by jacekbogdanski** [comment]
Hey @sryarama, so far we have no progress on this issue due to other priorities in CKEditor 4.

If you looking for a way to push this issue forward, you may contact our [support team](https://ckeditor.com/contact/) to see if there are any options to speed things up.

**Comment by sryarama** [comment]
This won't work in my scenario as my customers have no knowledge of CSS. If you can provide a tentative date or milestone information where this will be considered would be helpful.
